overlay/files-ubuntu-arm-rpi/system/oem/11_persistency.yaml

```diff
@@ -0,0 +1,37 @@
+name: "Configure persistent dirs bind-mounts"
+stages:
+  rootfs.after:
+    - if: '[ ! -f "/run/cos/recovery_mode" ]'
+      name: "Layout configuration"
+      environment_file: /run/cos/cos-layout.env
+      environment:
+        VOLUMES: "LABEL=COS_PERSISTENT:/usr/local"
+        OVERLAY: "tmpfs:25%"
+        RW_PATHS: "/var /etc /srv"
+        PERSISTENT_STATE_PATHS: >-
+          /etc/systemd
+          /etc/rancher
+          /etc/sysconfig
+          /etc/runlevels
+          /etc/ssh
+          /etc/iscsi 
+          /etc/cni
+          /home
+          /opt
+          /root
+          /usr/libexec
+          /var/log
+          /var/lib/rancher
+          /var/lib/kubelet
+          /var/lib/wicked
+          /var/lib/longhorn
+          /var/lib/cni
```

Comment:
We don't need to deal with this in this PR, IMO, but we will at some point: If a user wants to change something under `/boot/firmware/usercfg.txt`, that needs to be persisted across reboots.  With that said, we also probably want it to be read-only typically, so we need some way to configure it, and changes need to trigger a reboot.

Reply:
this can be customized by the user now directly from cloud config: #210  (not released yet, works only on master builds)